e2/src/test/scala/io/prediction/e2/evaluation/CrossValidationTest.scala

```diff
@@ -0,0 +1,124 @@
+package io.prediction.e2.evaluation
+
+import io.prediction.e2.evaluation.CrossValidationTest.EmptyEvaluationParams
```

Comment:
Probably don't need this as you have `CrossValidationTest.EmptyEvaluationParams()` in the code